### PR TITLE
Added a simple .wants command

### PR DESCRIPTION
**Pull request by zergling-man**
Which shows who has affinity to you. Separated by which ones you own. No longer includes my token...
Also indenting is whatever, it's not Python.

### CODE REVIEW

**Comment by Kwoth** [comment]
Can it get renamed to .waifuwants

**Comment by zergling-man** [comment]
Is the alias fine?

**Comment by Macley-Kun** [comment]
*bump*

**Comment by Macley-Kun** [comment]
@Kwoth Any comments on this pull 👀 before were getting some upstream stuff...

**Comment by shivaco** [comment]
Any news on this one? :thinking:

**Comment by zergling-man** [comment]
What hasn't (until now) made it into this thread is @Kwoth's comments along the lines of "I don't see the point of having a new command for this". There was a brief consideration about adding it to `.waifuinfo`, though even that's apparently going to be a hard sell.
I think this information is extremely useful, personally, but I haven't come up with any better way of including it than the waifuinfo addition. I suppose I should at least do that and then see.

File: NadekoBot.Core/Modules/Gambling/WaifuClaimCommands.cs [hunk 1]
```diff
@@ -471,6 +473,10 @@ public enum DivorceResult
                     await uow.CompleteAsync().ConfigureAwait(false);
                 }
 
+                // Dedupe, the right way.
+                affinities=affinities.Except(claims).ToList();
+                // Now it only contains unclaimed affinities.
```
Comment (Kwoth) [inline NadekoBot.Core/Modules/Gambling/WaifuClaimCommands.cs]: this should be a part of the query, not using linq

**Comment by zergling-man** [comment]
https://puu.sh/AwpbP.png - with an unclaimed affinity
https://puu.sh/Awpbw.png - with a claimed affinity. The * is not currently explained anywhere. I'm considering splitting that into two fields to be explicit about it, but I like the idea of it just showing up in the middle of the list. (The original intent of this is to allow you to see people you might want to claim, because they have affinity set. Seeing people you hold who have affinity or not is useful for considering divorces, but isn't something I explicitly care about supporting. Just figured it would be easy to do while I was at it.)

File: NadekoBot.Core/Modules/Gambling/WaifuClaimCommands.cs (hunk 1 above)
Comment (zergling-man) [reply NadekoBot.Core/Modules/Gambling/WaifuClaimCommands.cs]: You'll need to explain that in more detail to me. Keep in mind I'm learning not only Nadeko through field experience, but also dotnet.

**Comment by Kwoth** [comment]
Wouldn't it make more sense to put a 💞 for claimed person with affinity towards you

**Comment by Kwoth** [comment]
Can you change "Desires you" to "Loved by"

**Comment by zergling-man** [comment]
Oh, both of those are great ideas. I'll do that promptly.

**Comment by Macley-Kun** [comment]
@Kwoth 👀

**Comment by Kwoth** [comment]
I'll consider this in 3.0

**Pull request closed without merging**